client/blocks/reader-post-card/discover-byline.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { get } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import PostByline from './byline';
+import { getStreamUrl } from 'reader/route';
+
+class DiscoverPostByline extends PostByline {
+
+	static propTypes = {
+		post: React.PropTypes.object.isRequired,
+		site: React.PropTypes.object,
+		feed: React.PropTypes.object,
+		isDiscoverPost: React.PropTypes.bool,
+		showSiteName: React.PropTypes.bool,
+		originalPost: React.PropTypes.object,
+	}
+
+	getPostAuthor = () => {
+		// Will be null for site picks
+		return get( this.props.originalPost, 'author' );
+	}
+
+	getPostTimeLinkUrl = () => {
+		if ( this.props.originalPost ) {
+			return get( this.props.originalPost, 'URL' );
```

Comment:
We shouldn't use the original post for any external links. The discover permalink will redirect to the original. We use that track views in discover.